lib/runtime2/threepio/trace/TraceManager.ts

```diff
@@ -0,0 +1,97 @@
+import { TaskFlowSpan, SpanContext } from "./TaskFlowSpan";
+import { reportThreepioWarning } from "../utils/threepioLog";
+import util from 'util';
+
+export class TraceManager {
+  private static RootSpans: TaskFlowSpan[] = [];
+
+  public static getSpan(context: SpanContext): TaskFlowSpan {
+    const span = this.findSpanByRequestId(context.requestId);
+    if (span) {
+      this.updateMetadata(context);
+      return span;
+    } else {
+      return this.createSpan(context);
+    }
+  }
+
+  public static findSpanByRequestId(requestId: string, spans: TaskFlowSpan[] = this.RootSpans): TaskFlowSpan | undefined {
+    for (const span of spans) {
+      if (span.context.requestId === requestId) return span;
+      if (span?.children.length > 0) {
+        const found = this.findSpanByRequestId(requestId, span.children);
+        if (found) return found;
+      }
+    }
+    return undefined;
+  }
+
+  public static updateSpanMetric(requestId: string, key: string, value: number) {
+    const span = this.findSpanByRequestId(requestId);
+    if (span) {
+      span.metric(key, value);
+    }
+  }
+
+  public static updateMetadata(ctx: SpanContext) {
+    const span = this.findSpanByRequestId(ctx.requestId);
+    if (span) {
+      Object.entries(ctx.metadata).forEach(([key, value]) => {
+        span.context.metadata[key] = span.context.metadata[key]
+          ? [...(Array.isArray(span.context.metadata[key]) ? span.context.metadata[key] : [span.context.metadata[key]]), value]
+          : [value];
+      });
+    } else {
+      reportThreepioWarning(`TaskFlowSpan not found for requestId: ${ctx.requestId}`);
+    }
+  }
+
+  public static addError(requestId: string, err: Error) {
+    const span = this.findSpanByRequestId(requestId);
+    if (span) {
+      span.error(err);
+    }
+  }
+
+  public static endSpan(requestId: string) {
+    const span = this.findSpanByRequestId(requestId);
+    if (span) {
+      span.end();
+    }
+  }
+
+  public static clear() {
```

Comment:
```suggestion
  public static Clear() {
```
static 成员和方法都需要遵循需要首字母大写